src/couchapps/WMStats/_attachments/index.html

```diff
@@ -4,6 +4,9 @@
   <head>
     <title>WMStats</title>
     <style type="text/css" title="currentStyle">
+        @import "css/logdb.css";
+        @import "css/kube.min.css";
```

Comment:
Hi Valentin, This css file is missing from the head or patch? could you update that one?